### PR TITLE
Update auto merge time to 20 minutes

### PR DESCRIPTION
**Pull request by uilianries**
The CI build runs every 20 minutes now.

---

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by prince-chrismc — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by intelligide — DISMISSED** [review]

File: docs/review_process.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ It doesn't always mean accepting all the suggestions, but at least providing a r
 
 ## Automatic Merges
 
-The bot runs Automatic Merges every 30 minutes. Currently, it can only merge a single PR in this timeframe, so there is a theoretical limit of 48 PRs merged per day (in practice, it's even less for reasons listed below).
+The bot runs Automatic Merges every 20 minutes. Currently, it can only merge a single PR in this timeframe, so there is a theoretical limit of 48 PRs merged per day (in practice, it's even less for reasons listed below).
```
Comment (jgsogo) [inline docs/review_process.md:125]:
```suggestion
The bot runs Automatic Merges several times every hour. It merges a single PR in each run, so there is a limit of PRs merged per day. The frequency can be manually increased or decreased depending on the number of PRs that accumulate in the queue.
```

**Review by jgsogo — COMMENTED** [review]
I would vote to write something a bit more generic.

File: docs/review_process.md (hunk 1 above)
Comment (uilianries) [reply docs/review_process.md:125]: I prefer keep it explicit in terms of time. We have some users which ask the interval so we just need to point to this link. It's true we need to update it manually, but the information is transparent.

Comment (SSE4) [reply docs/review_process.md:125]: yes, explicit is better

**Comment by SSE4** [comment]
@uilianries btw, there are conflicts now :(

**Comment by uilianries** [comment]
> @uilianries btw, there are conflicts now :(

Done

**Review by SSE4 — APPROVED** [review]
thanks! apologize for invalidated earlier reviews

**Review by intelligide — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]

**Pull request merged**